modules/cloud-conf-shiftstack-reference.adoc

```diff
@@ -0,0 +1,123 @@
+//  Module included in the following assemblies:
+
+//  * installing/installing_openstack/installing-openstack-cloud-config-reference.adoc
+
+:_content-type: REFERENCE
+[id="cloud-conf-shiftstack-reference_{context}"]
+= OpenStack cloud provider options 
+
+The cloud provider configuration, typically stored as a file named `cloud.conf` controls how OpenShift Container Platform interacts with Red Hat OpenStack Platform (RHOSP).
+
+You can create a valid `cloud.conf` file by specifying the following options in it.
+
+[id="ccm-config-global-options"]
+== Global options
+
+The following options are used for {rh-openstack} CCM authentication with the {rh-openstack} Identity service, also known as Keystone. They are similiar to the global options that you can set by using the `openstack` CLI.
+
+|===
+| Option | Description
+
+| `auth-url`
+| The {rh-openstack} Identity service URL. For example, `http://128.110.154.166/identity`.
+
+| `ca-file`
+| Optional. The CA certificate bundle file for communication with the {rh-openstack} Identity service. If you use the HTTPS protocol with The Identity service URL, this option is required.
+
+| `domain-id`
+| The Identity service user domain ID. 
+
+ Leave this option unset if you are using Identity service application credentials.
+
+| `domain-name`
+| The Identity service user domain name. 
+
+ This option is not required if you set `domain-id`.
+
+| `tenant-id`
+| The Identity service project ID. Leave this option unset if you are using Identity service application credentials.
+
+In version 3 of the Identity API, which changed the identifier `tenant` to `project`, the value of `tenant-id` is automatically mapped to the project construct in the API. 
+
+| `tenant-name`
+| The Identity service project name. 
+
+
+| `username`
+| The Identity service user name. 
+
+Leave this option unset if you are using Identity service application credentials.
+
+| `password`
+| The Identity service user password. 
+
+ Leave this option unset if you are using Identity service application credentials.
+
+| `region`
+| The Identity service region name.
+
+| `trust-id`
+| The Identity service trust ID. A trust represents the authorization of a user, or trustor, to delegate roles to another user, or trustee. Optionally, a trust authorizes the trustee to impersonate the trustor. You can find available trusts by querying the `/v3/OS-TRUST/trusts` endpoint of the Identity service API.
+|===
+
+[id="ccm-config-lb-options"]
+== Load balancer options
+
+The cloud provider supports several load balancer options for deployments that use Octavia. 
+
+|===
+| Option | Description
+
+| `use-octavia`
+| Whether or not to use Octavia for the `LoadBalancer` type of service implementation rather than Neutron-LBaaS. The default value is `true`.
```

Comment:
```suggestion
| Whether or not to use Octavia for the `LoadBalancer` type of the service implementation rather than Neutron-LBaaS. The default value is `true`.
```